architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -228,32 +269,57 @@ metadata:
   uid: ee2b9405-c68c-11e5-8a99-525400f25e34
 spec: {}
 status:
-  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample
+  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample <2>
   tags:
   - items:
     - created: 2016-01-29T13:40:11Z
-      dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
+      dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d <3>
       generation: 1
-      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
-    tag: latest
+      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d <3>
+    tag: latest <4>
 ----
+
+<1> The name of the image stream. 
+<2> The image source repository.
+<3> The the internal registry service IP address, which is the target for pushes to the image stream. This valueis set automatically.
+<3> The SHA identifier that this image stream tag currently references. Resources that reference this image stream tag use this identifier.
+<4> The image stream tag.
+
 ====
 
+
+Using image streams has several significant benefits:
+
+* You can tag, rollback a tag, and quickly deal with images, without having to re-push using the command line.
+* You can trigger Builds and Deployments when new image is pushed to the registry.
+* You can xref:../../dev_guide/managing_images.adoc#importing-tag-and-image-metadata[mark a tag for a periodic re-import] on a defined schedule. If the source image has changed, that change is picked up and reflected in image stream and can trigger the Build and/or Deployment flow, depending upon the Build/Deployment configuration.
```

Comment:
I'd drop `on a defined schedule` entirely (even though I might be the one who originally suggested that change). Reading it over and over again suggests me that I (as a user) can define that schedule, but that's not true, so it's better not to mention anything about it. Just say it'll be done periodically, I think it's sufficient.